llvm/lib/CodeGen/GlobalISel/CombinerHelper.cpp

```diff
@@ -2645,6 +2645,14 @@ void CombinerHelper::replaceInstWithConstant(MachineInstr &MI, APInt C) {
   MI.eraseFromParent();
 }
 
+void CombinerHelper::replaceInstWithFConstant(MachineInstr &MI, ConstantFP *CFP) {
+  assert(MI.getNumDefs() == 1 && "Expected only one def?");
+  APFloat C = CFP->getValueAPF();
+  Builder.setInstr(MI);
```

Comment:
We should really just fix the combiner to always set the insert point before the apply

Reply:
Agreed.